autoPyTorch/pipeline/components/base_choice.py

```diff
@@ -247,35 +249,37 @@ def _check_dataset_properties(self, dataset_properties: Dict[str, Any]) -> None:
         """
         assert isinstance(dataset_properties, dict), "dataset_properties must be a dictionary"
 
-    def _apply_search_space_update(self, name: str, new_value_range: Union[List, Tuple],
-                                   default_value: Union[int, float, str], log: bool = False) -> None:
+    def _apply_search_space_update(self, hyperparameter_search_space_update: HyperparameterSearchSpaceUpdate) -> None:
         """Allows the user to update a hyperparameter
 
-        Arguments:
+        Args:
             name {string} -- name of hyperparameter
```

Comment:
And this could use a parenthesis rather than curly brackets

Reply:
yeah I dont know why it is curly :P I'll fix this